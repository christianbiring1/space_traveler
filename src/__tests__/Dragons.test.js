import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Dragonsdisplay from '../pages/Dragons';
import store from '../redux/configureStore';

it('matches Dragon container', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Dragonsdisplay />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
