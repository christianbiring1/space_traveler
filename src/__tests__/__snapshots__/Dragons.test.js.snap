// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches Dragon container 1`] = `
<div>
  <div
    className="dragons-container"
  />
</div>
`;
